utils/techutils/techutils.go

```diff
@@ -65,9 +65,26 @@ var TechToProjectType = map[Technology]project.ProjectType{
 	Dotnet: project.Dotnet,
 }
 
+var packageTypes = map[string]string{
+	"gav":      "Maven",
+	"docker":   "Docker",
+	"rpm":      "RPM",
+	"deb":      "Debian",
+	"nuget":    "NuGet",
+	"generic":  "Generic",
+	"npm":      "npm",
```

Comment:
what about yarn? poetry? conda? gradle?

Reply:
Also DotNet and Pnpm

Reply:
This code was just moved from diff place, did not changed this logic.
but for your information: some package managers considered the same so from the given information we can't deside what actually should be set, we just return the most common one

gav ==?> Gradle / Maven
nuget ==?> DotNet / Nuget
pypi ==?> Pip / Poetry / Venv

Reply:
but wouldn't we miss if we're "guessing" the most common one? isn't there a more accurate way here?